init.sh

```diff
@@ -31,6 +31,9 @@ if [ -n "$GDC_ENTRYPOINT" ]; then
   fi
 fi
 
+echo "Installing seek requirement..."
```

Comment:
Not sure what these requirements are for. But don't want them for every user I think. Put them in your project that you clone and install them. 
Delete everything under the `seek` folder. The GDC already mounts your ~/.aws/ files into the GDC if you specify `USE_AWS_HOME`.